src/Splunk.OpenTelemetry.AutoInstrumentation/nuget-readme.md

```diff
@@ -0,0 +1,82 @@
+# Using the Splunk.OpenTelemetry.AutoInstrumentation NuGet package
+
+## When to use the NuGet package
+
+Use the NuGet package in the following scenarios:
+
+1. Simplify deployment. For example, a container running a single application.
+1. Support instrumentation of [`self-contained`](https://learn.microsoft.com/en-us/dotnet/core/deploying/#publish-self-contained)
+  applications.
+1. Facilitate developer experimentation with automatic instrumentation through
+  NuGet packages.
+1. Solve version conflicts between the dependencies used by the application and the
+  automatic instrumentation.
+
+## Limitations
+
+While NuGet packages are a convenient way to deploy automatic
+instrumentation, they can't be used in all cases. The most common
+reasons for not using NuGet packages include the following:
+
+1. You can't add the package to the application project. For example,
+the application is from a third party that can't add the package.
+1. Reduce disk usage, or the size of a virtual machine, when multiple applications
+to be instrumented are installed in a single machine. In this case you can use
+a single deployment for all .NET applications running on the machine.
+1. A legacy application that can't be migrated to the [SDK-style project](https://learn.microsoft.com/en-us/nuget/resources/check-project-format#check-the-project-format).
```

Comment:
How common are these scenarios among our users? If not that much, it promotes the idea of turning this method into the main one... Otherwise, we'll need stronger reasons for not using NuGets in all cases.

Reply:
The most common case here are ASP.NET apps. So they are all on Windows and typically with IIS, for which our deployment via PowerShell/MSI is preferrable since typically there are multiple IIS apps in the same host.